src/thing.cpp

```diff
@@ -17,8 +17,6 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include "otpch.h"
```

Comment:
Yes atleast from my findings.
If you don't include otpch you will get an error about it not being included (if you use things like uintX_t etc).